### ISSUE TITLE
Pretrained model of AudiovisualEmotionLearner 

### ISSUE THREAD

**Issue by georgiamichou** [opendr-eu/opendr#506]
Which actors from the RAVDESS dataset were utilized for training the provided model of the AudiovisualEmotionLearner?
Based solely on the documentation, the provided information is as follows:

"This method is used to download a pretrained model from a given directory. Pretrained models are provided for the 'ia' fusion with 'zerodrop' modality dropout. The pretrained model is trained on RAVDESS dataset which is under CC BY-NC-SA 4.0 license."

**Comment by passalis** [comment]
@minhquoc0712 @katerynaCh Can you please check this. Thank you.
